test/playback.test.js

```diff
@@ -237,7 +237,7 @@ QUnit.test('DASH sidx', function(assert) {
 
     // Firefox sometimes won't loop if seeking directly to the duration, or to too close
     // to the duration (e.g., 10ms from duration). 100ms seems to work.
-    player.currentTime(player.duration() - 0.1);
+    player.currentTime(player.duration() - 0.5);
```

Comment:
are we having problems with 0.1?

Reply:
I was seing failures in firefox with this number every few runs, due to the increase in playbackRate.